scripts/reduce_CG1D.py

```diff
@@ -90,8 +91,10 @@ def main(inputfile: Union[str, Path], outputdir: Union[str, Path]) -> int:
 
     # step 2: extract info from inputfile
     update_dict = extract_info_from_path(str(inputfile))
+    assert update_dict["instrument"] == "CG1D", "Instrument is not CG1D"
```

Comment:
@gecage952  I'm looking at lines 93 to 98:
```python
    update_dict = extract_info_from_path(str(inputfile))
    assert update_dict["instrument"] == "CG1D", "Instrument is not CG1D"

    # step 3: update the dict and save dict to disk
    substitute_template(config_dict, update_dict)
    config_dict.update(update_dict)
```

They should be:
```python
    update_dict = extract_info_from_path(str(inputfile))
    assert update_dict["instrument"] == "CG1D", "Instrument is not CG1D"
    update_dict["outputdir"] = str(outputdir)

    # step 3: update the dict and save dict to disk
    config_dict = substitute_template(config_dict, update_dict)
```

Reply:
@gecage952 does this fixes the problem with `template.safe_substitute(**values)` ?

Reply:
Yes, that fixed all the issues.